dotcom-rendering/src/components/RelativeTime.importable.tsx

```diff
@@ -10,8 +10,8 @@ type Props = {
 };
 
 const ONE_MINUTE = 60_000;
-
-const getTime = () => Math.ceil(Date.now() / ONE_MINUTE) * ONE_MINUTE;
+const getNextMinute = (then: number) =>
+	then + Math.floor((Date.now() - then) / ONE_MINUTE) * ONE_MINUTE;
```

Comment:
Ensure that the time is either “now” or “1m ago”

Reply:
This rounds _down_ the current minute? In which case the naming of nextMinute is possibly confusing. Maybe:

`getCurrentMinute`?

Reply:
It can sometimes round up, but I think `getCurrentMinute` is clearer!

As way of an example, when `then` is 00:34: 
- “now” is 00:15, set to 00:34 ⬆️ 
- “now” is 00:50, set to 00:34 ⬇️ 
- “now” is 01:20, set to 00:34 ⬇️ 
- “now” is 01:34, set to 01:34 ➡️ 
- “now” is 01:50, set to 01:34 ⬇️ 
- “now” is 12:50, set to 12:34 ⬇️

Reply:
Could this be a JSDoc and/or a unit test to help understanding?